### ISSUE TITLE
dbld does not work with podman (Docker replacement) any more

### ISSUE THREAD

**Issue by czanik** [syslog-ng/syslog-ng#3744]
For the past couple of years I used Podman / Buildah with dbld to build syslog-ng release tgz and check if CentOS 7 build still works. Podman is a Docker replacement, and normally there are no compatibility problems. 

First I realized the problem soon before the 3.33.1 release, but as I also have Docker on the host, I did not care much. Now I tested again, and it still fails with the latest git master snapshot I did this morning:

```
leap152:~ # rm -fr syslog-ng/
leap152:~ # git clone https://github.com/syslog-ng/syslog-ng/
Cloning into 'syslog-ng'...
remote: Enumerating objects: 97962, done.
remote: Counting objects: 100% (1867/1867), done.
remote: Compressing objects: 100% (794/794), done.
remote: Total 97962 (delta 1169), reused 1467 (delta 1037), pack-reused 96095
Receiving objects: 100% (97962/97962), 30.49 MiB | 10.72 MiB/s, done.
Resolving deltas: 100% (70486/70486), done.
leap152:~ # cd syslog-ng/
leap152:~/syslog-ng # cp ../rules.conf dbld/
leap152:~/syslog-ng # cat ../rules.conf 
DOCKER=podman

leap152:~/syslog-ng # dbld/rules rpm-centos-7
Checking /root/syslog-ng/dbld/build/syslog-ng-3.33.2.21.g1c31487.tar.gz if it is up-to-date...
Rebuilding, because tarball /root/syslog-ng/dbld/build/syslog-ng-3.33.2.21.g1c31487.tar.gz not found
Git status follows:
[...]
tardir=syslog-ng-3.33.2.21.g1c31487 && tar --format=ustar -chf - "$tardir" | eval GZIP= gzip --best -c >syslog-ng-3.33.2.21.g1c31487.tar.gz
if test -d "syslog-ng-3.33.2.21.g1c31487"; then find "syslog-ng-3.33.2.21.g1c31487" -type d ! -perm -200 -exec chmod u+w {} ';' && rm -rf "syslog-ng-3.33.2.21.g1c31487" || { sleep 5 && rm -rf "syslog-ng-3.33.2.21.g1c31487"; }; else :; fi
Your tarball is in /build, also available on the host in $(top_srcdir)/dbld/build
Generating snapshot version in debian/
Generating snapshot version in syslog-ng.spec
podman run -e USER_NAME_ON_HOST=root --network=host --privileged --ulimit nofile=1024:1024 -v /root/syslog-ng:/source -v /root/syslog-ng/dbld:/dbld -v /root/syslog-ng/dbld/build:/build -v /root/syslog-ng/dbld/install:/install -e CONFIGURE_OPTS="${CONFIGURE_OPTS:---enable-debug --enable-manpages --with-python=2 --prefix=/install }" -e CCACHE_DIR=/build/ccache -e MODE=snapshot -e VERSION=3.33.2.21.g1c31487 -e PATH=/usr/lib/ccache:/usr/lib64/ccache:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin -e GRADLE_USER_HOME=/build/gradle-home -e GRADLE_PROJECT_CACHE_DIR=/build/gradle-cache -e GRADLE_FLAGS=--build-cache  --rm -i  balabit/syslog-ng-centos-7 /dbld/rpm
rm: cannot remove '/build/': Device or resource busy
make: *** [dbld/rules:153: rpm-centos-7] Error 1
leap152:~/syslog-ng # 
```
RHEL / CentOS 8 and recent Fedora releases do not have Docker any more.

**Comment by szemere** [comment]
TL;DR : refreshing the images solved the problem. `podman pull balabit/syslog-ng-centos-7`

In details:
KUDO for @czanik for the live debugging.

- The error message `rm: cannot remove '/build/': Device or resource busy` were triggered by this line: https://github.com/syslog-ng/syslog-ng/blob/master/dbld/rpm#L10
- The problem was that the `${IMAGE_PLATFORM}` environment variable was empty, and we tried to execute `rm -rf /build/` instead of `rm -rf /build/centos-7`.
- The name of the environment variable were changed here: https://github.com/syslog-ng/syslog-ng/commit/d60cc7ca209b19b6b1f2f10667ad8a0d7da8ba35
- "Unfortunately" docker/podman do not emit an error message because of a missing environment variable. So the error went silent.
